commitlint.config.ts

```diff
@@ -0,0 +1,37 @@
+import { RuleConfigSeverity } from "@commitlint/types";
+
+import type { UserConfig as CommitlintUserConfig } from "@commitlint/types";
+
+/**
+ * `commitlint` configuration.
+ * @see https://commitlint.js.org/#/reference-configuration
+ */
+const commitlintConfig: CommitlintUserConfig = {
```

Comment:
Unit tests will be added to test the CLI against this config (added to https://trello.com/c/O7zkMZIX/41-commit-linting-hook scope)